obs-server/obs_instance_configuration.xml

```diff
@@ -0,0 +1,31 @@
+<configuration>
+  <title>Debian LLVM OBS builder</title>
+  <description>  &lt;p class="description"&gt;
+		The Debian LLVM OBS builder is used to both distribute upstream LLVM .deb packages
```

Comment:
you can trim the description just to keep your part!

Reply:
Done